app/components/appropriate_bodies/participants/table_row.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module AppropriateBodies
+  module Participants
+    class TableRow < BaseComponent
+      with_collection_parameter :induction_record
+
+      delegate :teacher_profile,
+               :role,
+               to: :participant_profile,
+               allow_nil: true
+
+      delegate :full_name,
+               to: :user,
+               allow_nil: true
+
+      delegate :participant_profile,
+               :training_status,
+               :school,
+               :user,
+               :cohort,
+               :lead_provider_name,
+               :delivery_partner_name,
+               to: :induction_record,
+               allow_nil: true
+
+      def initialize(induction_record:)
+        @induction_record = induction_record
+      end
+
+      def status_tag
+        return unless status_name
+
+        title = t("participant_profile_status.status.#{status_name}.title")
+        description = t("participant_profile_status.status.#{status_name}.description")
+
+        if description.present?
+          content_tag(:strong, title) +
+            content_tag(:p, description, class: "govuk-body-s")
+        else
+          content_tag(:strong, title)
```

Comment:
Not really important here, but if you know the tag you can call it as a method directly. So instead of `content_tag(:strong, title)` it'd be `tag.strong(title)`.

Reply:
@peteryates 
Cool, will make those changes.